main.go

```diff
@@ -57,27 +58,78 @@ func _main() int {
 		go eniConfigController.Start()
 	}
 
-	awsK8sAgent, err := ipamd.New(discoverController, eniConfigController)
+	ipamContext, err := ipamd.New(discoverController, eniConfigController)
 
 	if err != nil {
-		log.Error("Initialization failure ", err)
+		log.Errorf("Initialization failure: %v", err)
 		return 1
 	}
 
 	// Pool manager
-	go awsK8sAgent.StartNodeIPPoolManager()
+	go ipamContext.StartNodeIPPoolManager()
 
 	// Prometheus metrics
-	go awsK8sAgent.ServeMetrics()
+	go ipamContext.ServeMetrics()
 
 	// CNI introspection endpoints
-	go awsK8sAgent.ServeIntrospection()
+	go ipamContext.ServeIntrospection()
 
-	err = awsK8sAgent.RunRPCHandler()
+	// Copy the CNI plugin and config. This will mark the node as Ready.
+	log.Info("Copying /app/aws-cni to /host/opt/cni/bin/aws-cni")
+	err = copyFileContents("/app/aws-cni", "/host/opt/cni/bin/aws-cni")
 	if err != nil {
-		log.Error("Failed to set up gRPC handler ", err)
+		log.Errorf("Failed to copy aws-cni: %v", err)
 		return 1
 	}
 
+	log.Info("Copying /app/10-aws.conflist to /host/etc/cni/net.d/10-aws.conflist")
```

Comment:
I think this is fine for now, but do we want to parameterize this filename? I'm take it or leave it